agent/agent.go

```diff
@@ -953,6 +963,13 @@ func (a *agent) updateCommandEnv(current []string) (updated []string, err error)
 		envs[k] = v
 	}
 
+	// Prepend the agent script bin directory to the PATH
+	// (this is where Coder modules place their binaries).
+	if _, ok := envs["PATH"]; !ok {
+		envs["PATH"] = os.Getenv("PATH")
+	}
+	envs["PATH"] = a.scriptRunner.ScriptBinDir() + ":" + envs["PATH"]
```

Comment:
I think in Windows you need to use `;` to separate path elements.

Reply:
Thanks for noticing! @mtojek no need, we can simply use `filepath.ListSeparator`.